.NET/Microsoft.Recognizers.Definitions/Japanese/NumbersWithUnitDefinitions.cs

```diff
@@ -0,0 +1,35 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+//     
+//     Generation parameters:
+//     - DataFilename: Patterns\Japanese\Japanese-NumbersWithUnit.yaml
+//     - Language: Japanese
+//     - ClassName: NumbersWithUnitDefinitions
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Microsoft.Recognizers.Definitions.Japanese
+{
+	using System;
+	using System.Collections.Generic;
+
+	public static class NumbersWithUnitDefinitions
+	{
+		public static readonly IList<string> AgeAmbiguousValues = new List<string>
+		{
+			"歳"
```

Comment:
Can you add a comment in the YAML file describing what this means in English? Or examples of the ambiguity?